doc/generic/pgf/CHANGELOG.md

```diff
@@ -0,0 +1,3294 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
+
+Even though this release is not too heavy on user-facing additions it has seen a
+lot of contributed changes. Thanks to everyone who volunteered their time!
+
+### BREAKING CHANGES
+
+- `\pgfversiondatetime` and `\pgfrevisiondatetime` have been removed.
+  `\pgfversiondate` and `\pgfrevisiondate` now use the format `YYYY-MM-DD`.
+  `\pgfrevision{,date}` and `\pgfversion{,date}` are synonyms for now, but the
+  revision should eventually gain back its original meaning.  However, as of now
+  this is not supported by l3build.
+
+### Added
+
+- l3build support for packaging PGF/TikZ
+- Add Matrix chat to README
+- Add rhombic polihedra #1022
+- Add Developer Certificate of Origin (DCO) to Pull Request template and enforce
+
+### Fixed
+
+- Wrap box content in `\pgfinterruptpicture` for shapes
+- Remove superfluous `;` for shape `arrow box`
+- Remove superfluous `/utils/exec` in animations
+- Gobble `\pgf@stop` when parsing finishes in animations
+- Add missing `\pgf@sys@tonumber` before `<dimen>` in drivers and animations
+- Rewrite `dash expand off`
+- Better unknown key error msg in decorations
+- Fix `\let` in drivers for two csnames #1088
+- Protect against comma in pgfkeys arguments #389
+- Let active `"` expand to non-active `"` in pgfmath #1062
+- Eagerly evaluate `/tikz/rotate fit` #1071
```

Comment:
It's noticeable that this also fixes the problem that `/tikz/rotate fit` avoids comma in its argument (a math expr). How about

```suggestion
- Protect against comma in `/tikz/rotate fit` argument and make it
  eagerly evaluated #1071
```